cmd/eksctl/create.go

```diff
@@ -188,6 +189,13 @@ func doCreateCluster(cfg *api.ClusterConfig, name string) error {
 			return err
 		}
 
+		// add default storage class
+		if cfg.Addons.Storage == true {
+			if err := ctl.AddDefaultStorageClass(clientSet); err != nil {
+				return err
+			}
+		}
+
```

Comment:
I might recommend moving this after the `NodeAuthConfigMap` creation. Even though this should work without any issues it feels a little odd to create this before the worker nodes are able to connect to the control plane… @errordeveloper @richardcase thoughts?

Reply:
"this" representing the full `AddDefaultStorageClass` block.

Reply:
Thanks @christopherhein!  I can definitely do that if y'all decide that's best.  Let me know @richardcase / @errordeveloper 

:)

Reply:
@christopherhein - yes i agree with you. I think it makes more logical sense.

Reply:
Awesome!  I just moved that new function after the nodes join.  Thank you so much!